sponsoredTransactions/frontend/Jenkinsfile

```
@@ -26,7 +26,8 @@ pipeline {
                       --label build_image="${build_image}" \
                       --tag="${image_name}" \
                       --pull \
-                      ./examples/sponsoredTransactions
+                      --file sponsoredTransactions/Dockerfile
+                      ./sponsoredTransactions
```

You said you needed some dependencies from the root, if they are needed in the docker build you need to send in the root as context
```suggestion
                      .
```